tutorial/go/fuzz/fuzz.go

```diff
@@ -0,0 +1,144 @@
+package thriftfuzz
```

Comment:
1. Is there a reason this package name is different from the directory name? I understand that this is not really some library code others would import, but this still seem unnecessary.
2. Is there a reason this is under tutorial instead of, say, `lib/go/test`?

Reply:
> Is there a reason this package name is different from the directory name?

Not really... Just wanted the name to be self-explaining

> Is there a reason this is under tutorial instead of, say, lib/go/test?

I was not sure where to put it.
Since it uses the tutorial, I put it close to it.
but `lib/go/test` seems a better choice. Moving there